src/predicates/in.jl

```diff
@@ -34,6 +34,31 @@ Base.in(p::Point, pl::Plane) = isapproxzero(udot(normal(pl), p - pl(0, 0)))
 
 Base.in(p::Point, b::Box) = minimum(b) ⪯ p ⪯ maximum(b)
 
+function lon_in_interval(lon::Deg, left::Deg, right::Deg)
+  return if left < 0 && right ≥ 0
```

Comment:
```suggestion
  if left < 0 && right ≥ 0
```
We don't use `return` in the last statement of the function.

Reply:
:heavy_check_mark: